to.etc.domui/src/main/java/to/etc/domui/component/upload/FileUploadMultiple.java

```diff
@@ -345,6 +347,9 @@ public boolean handleUploadRequest(@NonNull RequestContextImpl param, @NonNull C
 			if(uiar != null) {
 				for(UploadItem ui : uiar) {
 					m_value.add(ui);
+					if(m_isRegisterAutoCleanup){
+						conversation.registerTempFile(ui.getFile());
```

Comment:
getFile() can return null in which case it should not be passed to registerTempFile().